runtime/onert/backend/train/KernelGenerator.cc

```diff
@@ -418,16 +418,16 @@ void KernelGenerator::visit(const ir::train::operation::Loss &node)
     {
       const auto y_pred_op_code = node.y_pred_op_code();
       bool is_normalization_required = (y_pred_op_code != ir::OpCode::Softmax);
+      auto cce_params = std::get<ir::train::CategoricalCrossentropyParam>(loss_param);
```

Comment:
If loss_param holds `std::monostate`, then this code will throw a `std::bad_variant_access` exception at runtime.